hubstorage/serialization.py

```diff
@@ -16,6 +17,14 @@ def jldecode(lineiterable):
         yield loads(line)
 
 
+def mpdecode(iterable):
+    unpacker = Unpacker()
```

Comment:
Can we distinguish [binary and string types])https://pypi.python.org/pypi/msgpack-python#string-and-binary-type) from the start?